v3-nginx/Dockerfile

```diff
@@ -72,7 +74,10 @@ RUN apt-get update \
       libxml2-dev \
       make \
       wget \
-      zlib1g-dev
+      zlib1g-dev \
+      vim-tiny && \
```

Comment:
Do we need an editor in a productive WAF image?

Reply:
No, but it was there before :) We can get rid of that one.